README.md

```diff
@@ -64,6 +64,41 @@ Flags:
       --tiller-out-cluster       when Tiller is not running in the cluster e.g. Tillerless
 ```
 
+### Clean up Helm v2 data
+
+Clean up Helm v2 configuration, release data and Tiller deployment:
+
+```console
+$ helm  2to3 cleanup [flags]
+
+Flags:
+      --dry-run                  simulate a command
+  -h, --help                     help for cleanup
+  -l, --label string       label to select tiller resources by (default "OWNER=TILLER")
+  -s, --release-storage string   v2 release storage type/object. It can be 'secrets' or 'configmaps'. This is only used with the 'tiller-out-cluster' flag (default "secrets")
+  -t, --tiller-ns string         namespace of Tiller (default "kube-system")
+      --tiller-out-cluster       when  Tiller is not running in the cluster e.g. Tillerless
+```
+
+It will clean:
+- Configuration (Helm home directory)
+- v2 release data
+- Tiller deployment
+
+For cleanup it uses default Helm v2 home and v3 config and data folders.
```

Comment:
It should be only: For cleanup it uses default Helm v2 home folder.

Reply:
done